pkgs/by-name/ag/age-plugin-xwing/package.nix

```
@@ -0,0 +1,33 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "age-plugin-xwing";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "Rixxc";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-jjIArFXbFjhemBDbJdphhWfqID/UGmOxBUeKTIT+2jU=";
+  };
+
+  useFetchCargoVendor = true;
+  cargoHash = "sha256-54ZYd/wNHpm38KuMxUStXUiaz94pdqGEaM381zpsulg=";
+
+  doCheck = true;
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = with lib; {
+    description = "X-Wing plugin for age";
+    homepage = "https://github.com/Rixxc/age-plugin-xwing";
+    license = with licenses; [ mit ];
+    maintainers = with maintainers; [ rixxc ];
+    mainProgram = "age-plugin-xwing";
+  };
```

```suggestion
  meta = {
    description = "X-Wing plugin for age";
    homepage = "https://github.com/Rixxc/age-plugin-xwing";
    license = with lib.licenses; [ mit ];
    maintainers = with lib.maintainers; [ rixxc ];
    mainProgram = "age-plugin-xwing";
  };
```

Please try to avoid using large scopes like 'with lib,' as they may cause confusion